code/modules/clothing/factions/gezena.dm

```diff
@@ -86,12 +86,14 @@
 	desc = "Raksha - a Kalixcian word for 'protection of the heart'. Sturdy and reliable."
 	icon_state = "marinevest"
 	item_state = "marinevest"
+	armor = list("melee" = 35, "bullet" = 50, "laser" = 50, "energy" = 50, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50) //the laser gun country should probably have laser armor
```

Comment:
Thats quite a lot of armor in total

Reply:
I accidentally deleted my draft comment because groggy but tldr;

Bulletproof armor has 60 bullet armor
Ablative armor has 60 laser
Regular has 35 melee

Lorewise it makes sense for laser centric planet to have good laser armor
Since frontier is one of the few places that use ballistics, they also equip frontier units with ballistic armor

Gameplay wise it's so the pgf can interact better with the gunplay balancing, but also you can just fucking shoot the arms/legs which aren't protected, just like the Regular bulletproo/Ablative armor